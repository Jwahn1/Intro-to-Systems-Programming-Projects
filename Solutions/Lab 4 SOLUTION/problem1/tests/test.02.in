2
2 Bagels 1.25
3 Bagels 1.25
