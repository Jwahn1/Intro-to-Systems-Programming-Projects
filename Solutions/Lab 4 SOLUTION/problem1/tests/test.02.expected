1
5 Bagels 1.25
